upstairs/src/downstairs.rs

```diff
@@ -3151,7 +3151,7 @@ impl Downstairs {
         let Some(job) = self.ds_active.get_mut(&ds_id) else {
             error!(
                 self.clients[client_id].log,
-                "IO completion error: missing {ds_id} "
+                "[{client_id}] IO completion error: missing {ds_id} "
```

Comment:
This should already be in the logs: we're writing to `self.clients[client_id].log`, which has a `client => id` tag.